libvast/vast/format/mrt.hpp

```diff
@@ -503,13 +501,18 @@ struct message_as4_parser : parser<message_as4_parser> {
 
 } // namespace bgp4mp
 
+struct epsilon{
+  uint32_t length;
+};
```

Comment:
Do you have any plans to use the `length` field here? If not, then I'd suggest to simply put `vast::none` in the first variant type. (First, because this is the type that gets default-constructed.)